src/Components/Facility/CoverImageEditModal.tsx

```diff
@@ -107,120 +154,326 @@ const CoverImageEditModal = ({
+                    ? "border-red-500"
+                    : "border-gray-500"
                 }`}
               >
-                <path d="M28 8H12a4 4 0 0 0-4 4v20m32-12v8m0 0v8a4 4 0 0 1-4 4H12a4 4 0 0 1-4-4v-4m32-4-3.172-3.172a4 4 0 0 0-5.656 0L28 28M8 32l9.172-9.172a4 4 0 0 1 5.656 0L28 28m0 0 4 4m4-24h8m-4-4v8m-12 4h.02" />
-              </svg>
-              <p
-                className={`text-sm ${
-                  dragProps.dragOver && "text-primary-500"
-                } ${
-                  dragProps.fileDropError !== ""
-                    ? "text-red-500"
-                    : "text-gray-700"
-                } text-center`}
+                <svg
+                  stroke="currentColor"
+                  fill="none"
+                  viewBox="0 0 48 48"
+                  aria-hidden="true"
+                  className={`w-12 h-12 stroke-[2px] ${
+                    dragProps.dragOver && "text-primary-500"
+                  } ${
+                    dragProps.fileDropError !== ""
+                      ? "text-red-500"
+                      : "text-gray-600"
+                  }`}
+                >
+                  <path d="M28 8H12a4 4 0 0 0-4 4v20m32-12v8m0 0v8a4 4 0 0 1-4 4H12a4 4 0 0 1-4-4v-4m32-4-3.172-3.172a4 4 0 0 0-5.656 0L28 28M8 32l9.172-9.172a4 4 0 0 1 5.656 0L28 28m0 0 4 4m4-24h8m-4-4v8m-12 4h.02" />
+                </svg>
+                <p
+                  className={`text-sm ${
+                    dragProps.dragOver && "text-primary-500"
+                  } ${
+                    dragProps.fileDropError !== ""
+                      ? "text-red-500"
+                      : "text-gray-700"
+                  } text-center`}
+                >
+                  {dragProps.fileDropError !== ""
+                    ? dragProps.fileDropError
+                    : "Drag & drop image to upload"}
+                </p>
+                <p className="mt-4 text-gray-700 font-medium text-center">
+                  No cover photo uploaded for this facility. <br />
+                  Recommended aspect ratio for facility cover photo is 1:1.
+                </p>
+              </div>
+            )}
+
+            <div className="flex flex-col sm:flex-row p-4 gap-2">
+              <div>
+                <label className="rounded-lg bg-white py-2 px-4 text-primary-500 font-medium border border-primary-500 hover:text-primary-400 hover:border-primary-400 text-sm flex gap-1 items-center justify-center cursor-pointer transition-all">
+                  <i className="fas fa-cloud-upload-alt mr-2"></i>Upload an
+                  image
+                  <input
+                    title="changeFile"
+                    type="file"
+                    accept="image/*"
+                    className="hidden"
+                    onChange={onSelectFile}
+                  />
+                </label>
+              </div>
+              <div className="sm:flex-1" />
+              <button
```

Comment:
Use `ButtonV2`

Reply:
We can migrate the button in some other issue as it is not related to the feature @rithviknishad

Reply:
as input tag is involved while using buttonv2 the functionality is not achived hence i have used the normal button here